fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/domain/Report.java

```diff
@@ -111,7 +111,7 @@ public static Report fromJson(final JsonCommand command, final Collection<String
         return new Report(reportName, reportType, reportSubType, reportCategory, description, useReport, reportSql, reportTypes);
     }
 
-    protected Report() {
+    Report() {
```

Comment:
@percyashu couldn't this just be removed, instead of made package local?

Reply:
Sure will remove them

Reply:
I look at these and JPA requires entities to have a no-args constructo .